src/Forms/Shared/Samples/Data/CreateMobileGeodatabase/CreateMobileGeodatabase.xaml.cs

```diff
@@ -0,0 +1,219 @@
+                tableDescription.FieldDescriptions.Add(new FieldDescription("oid", FieldType.OID));
+                tableDescription.FieldDescriptions.Add(new FieldDescription("collection_timestamp", FieldType.Date));
+
+                // Add a new table to the geodatabase by creating one from the table description.
+                _featureTable = await _geodatabase.CreateTableAsync(tableDescription);
+
+                // Clear the table.
+                await UpdateTable();
+
+                // Create a feature layer for the map.
+                FeatureLayer featureLayer = new FeatureLayer(_featureTable);
+                MyMapView.Map.OperationalLayers.Add(featureLayer);
+
+                // Add an event handler for new feature taps.
+                MyMapView.GeoViewTapped += MyMapView_GeoViewTapped;
+            }
+            catch (Exception ex)
+            {
+                await Application.Current.MainPage.DisplayAlert(ex.GetType().Name, ex.Message, "OK");
+            }
+        }
+
+        private void MyMapView_GeoViewTapped(object sender, Esri.ArcGISRuntime.Xamarin.Forms.GeoViewInputEventArgs e)
+        {
+            _ = AddFeature(e.Location);
+        }
+
+        private async Task AddFeature(MapPoint location)
+        {
+            // Create an attributes dictionary for the feature.
+            var attributes = new Dictionary<string, object>();
+
+            // Add a timestamp of when the feature was created.
+            attributes["collection_timestamp"] = DateTime.Now;
+
+            // Create the feature.
+            Feature feature = _featureTable.CreateFeature(attributes, location);
+            try
+            {
+                // Add the feature to the feature table.
+                await _featureTable.AddFeatureAsync(feature);
+
+                // Update the UI.
+                await UpdateTable();
+                FeaturesLabel.Text = $"Number of features added: {_featureTable.NumberOfFeatures}";
+            }
+            catch (Exception ex)
+            {
+                await Application.Current.MainPage.DisplayAlert(ex.GetType().Name, ex.Message, "OK");
+            }
+        }
+
+        private async Task UpdateTable()
+        {
+            // Query all of the features in the feature table.
+            FeatureQueryResult queryFeatureResult = await _featureTable.QueryFeaturesAsync(new QueryParameters());
+
+            // Set the items source for the data grid with the updated query result.
+            FeatureListView.ItemsSource = queryFeatureResult;
+        }
+
+        private void ViewTable(object sender, EventArgs e)
+        {
+            TableFrame.IsVisible = true;
+        }
+
+        private void CloseTable(object sender, EventArgs e)
+        {
+            TableFrame.IsVisible = false;
+        }
+
+        private void CloseGeodatabaseClick(object sender, EventArgs e)
```

Comment:
I think this sample may be a good case for using regions to separate the event handlers from the table manipulation methods.

Reply:
Closing the geodatabase is important here, I'd rather not hide it in a region in `CloseGeodatabaseClick` or `SampleUnloaded`.